### ISSUE TITLE
ePSXe auto-save/load bug

### ISSUE THREAD

**Issue by ReCoJav** [ResetCollection/support#78]
With Android's ePSXe, I have it set to autosave when the app closes, minimizes (android home screen is pressed), or the phone screen is turned off. Normally, when restarting the app it prompts you to restart the autosave profile. Whatever you choose, the autosave file deletes.

When I open the app through Reset, if this prompts, it doesn't load. Instead, the screen "flashes," then repeats the prompt. If I open ePSXe directly after this happens, the autosave is already deleted, which seems normal since you have only the one chance to recover the data. The only way around this is to open ePSXe directly after minimizing Reset, or else my progress is loss.

TL:DR - ePSXe autosave on close feature fails to load if opened via Reset.

From: https://www.reddit.com/r/ResetCollection/comments/gcshpt/android_epsxe_bug/
